tools/evm2wasm/CMakeLists.txt

```diff
@@ -1,2 +1,5 @@
 add_executable(evm2wasm main.cpp)
-target_link_libraries(evm2wasm PRIVATE libevm2wasm)
\ No newline at end of file
+
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_package(Threads REQUIRED)
+target_link_libraries(evm2wasm PRIVATE libevm2wasm Threads::Threads)
```

Comment:
Why does it neads pthreads? Under no circumstances should it use threads :)

Reply:
Binaryen was complaining.  See the error message above.

Reply:
That is weird though, we never get that with Hera.

Reply:
Btw, this was actually fixed in Hera since.